dbhelper.js

```diff
@@ -0,0 +1,47 @@
+const knex = require('knex');
+
+const db_config = require('./knexfile');
+
+const db = knex(db_config.development);
+
+getDishes = () => {
+    return db('dishes')
+};
+
+
+addDish = (dish) => {
+    return db('dishes')
+        .insert(dish)
+        .then(ids => ({ id: ids[0] })
```

Comment:
Your `insert` method here returns an array of the ids of any newly added dishes. In this case, you pretty much expect/assume it to be a single id (meaning one dish was added), so that's why we take the first id. The reason we have the parens is so that JS doesn't read our object as a function body. Without those, it'd have to look like this:
```javascript
.then(ids => {
  return { id: ids[0] };
})
```
one thing to note is that you could use [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring) in order to make this a very slightly smoother:
```javascript
// (([ id ])) takes off the first element of the array and names it `id`
// ({ id }) sends an object with a key named `id` 
//     with the value of our destructured `id` param
.then(([ id ]) => ({ id }))
```